.gitmodules

```diff
@@ -18,7 +18,7 @@
 	url = https://github.com/meta-rust/meta-rust.git
 [submodule "layers/meta-up-board"]
 	path = layers/meta-up-board
-	url = https://github.com/AaeonCM/meta-up-board.git
+	url = https://github.com/up-division/meta-up-board.git
```

Comment:
Please also leave a note here that the switch to the new BSP repo has been advised by AAEON themselves